projects/kit/components/input-date-range/input-date-range.component.ts

```diff
@@ -312,6 +312,10 @@ export class TuiInputDateRangeComponent
         this.toggle();
         this.focusInput();
 
+        if (!range) {
```

Comment:
Don't need to add tests for this? ))

Reply:
I don't think so, this is a change detection problem, the unit test will be useless (